### DISCUSSION TITLE
Add the ability to add records nested two levels deep

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#596] [Feature requests]
Given that you have the following models and associations (with resources attached) `Post` which `has_one` `SlideShow` which in term `has_many` `SlideShowItems`, you'd want to have the ability to add `SlideShowItems` from the `Post` show screen directly.

**Comment by JensRavens** [comment]
I'd love Avo in all future projects with my agency, but the missing support for nested fields is kind of a dealbreaker for us as we quite often deal with pretty deep hierarchies of objects. Let me know if there is anything we can do to help :)

**Reply by adrianthedev** [reply]
Hey @JensRavens, thanks for writing on this. We'd love to make this work for you on the bigger scale.

Would you like to have a small chat with me so I can better understand your use cases?
I promise this is for actual research that goes into building a better Avo and won't waste your time.

**Reply by JensRavens** [reply]
Sure, just tell me what you want to know :)

**Reply by adrianthedev** [reply]
I'd love to see exactly how you implemented nested fields in past apps (what kinds of associations, how deeply nested, what information you expose, what updates you're bringing, etc.) to get a feel on how to design the API.

We might communicate a bit better over a 10-15 mins [video call](https://whereby.com/adrianthedev). I'm free today for the next 3 hours but I'll also leave my calendly link if you have different availability.

**Reply by JensRavens** [reply]
Today I'm a bit short on time, but tomorrow I'll be free after 14:30 CET. Here's my email: jens@nerdgeschoss.de

**Reply by adrianthedev** [reply]
Done! Thank you Jens. Talk to you tmr.
